src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IUsingStatement.vb

```diff
@@ -80,48 +80,59 @@ BC30201: Expression expected.
             Dim node = tree.GetRoot().DescendantNodes().OfType(Of UsingBlockSyntax).Single()
             Dim op = DirectCast(comp.GetSemanticModel(tree).GetOperationInternal(node), IUsingStatement)
 
-            Assert.Null(op.Declaration)
+            Assert.Equal(OperationKind.InvalidExpression, op.Expression.Kind)
         End Sub
 
         <CompilerTrait(CompilerFeature.IOperation)>
         <Fact>
-        <WorkItem(19887, "https://github.com/dotnet/roslyn/issues/19887")>
-        Public Sub UsingDeclarationExistingVariableNullDeclaration()
-            Dim source = <compilation>
-                             <file name="c.vb">
-                                 <![CDATA[
+        Public Sub IUsingStatement_MultipleNewResources()
+            Dim source = <![CDATA[
 Imports System
-Module Module1
-    Class C1
+
+Module Program
+    Class C
         Implements IDisposable
         Public Sub Dispose() Implements IDisposable.Dispose
-            Throw New NotImplementedException()
         End Sub
     End Class
-    Sub S1()
-        Dim x = New C1()
-        Using x
+    Sub Main(args As String())
+        Using c1 As C = New C, c2 As C = New C'BIND:"Using c1 As C = New C, c2 As C = New C"
+            Console.WriteLine(c1)
         End Using
     End Sub
-End Module
-]]>
-                             </file>
-                         </compilation>
+End Module]]>.Value
 
-            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(source, parseOptions:=TestOptions.RegularWithIOperationFeature)
-            CompilationUtils.AssertNoDiagnostics(comp)
+            Dim expectedOperationTree = <![CDATA[
+IUsingStatement (OperationKind.UsingStatement) (Syntax: 'Using c1 As ... End Using')
+  Expression: 
+    IVariableDeclarationStatement (0 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Using c1 As ... s C = New C')
```

Comment:
Should this be `2 declarations`?

Reply:
VB Using declarations are currently broken, @heejaechang has a PR out to fix this here:  https://github.com/dotnet/roslyn/pull/22570